src/App.less

```diff
@@ -5,6 +5,10 @@
     overflow-x: hidden;
 }
 
+body when (@theme = 'dark') {
```

Comment:
Ah I noticed this change after I merged, is this needed or was it added by mistake?

Reply:
Oh no, my mistake. Sorry. This needs to be removed. If you shortly can do this directly on the master. I would be glad.

Reply:
Cool! Fixed on master: https://github.com/caprover/caprover-frontend/commit/42aecf94702de17e03738051b30e1b7165a171b8

Reply:
Cool! Thanks :)